azure-pipelines.yml

```diff
@@ -38,13 +38,13 @@ jobs:
       yapf --version
       isort --version
       clang-format -version
+      flake8 --version
       echo "PATH=$PATH"
     displayName: Display tool versions
   - bash: |
-      find util -iname '*.py' -print0 \
-        | xargs -0 -n1 $PWD/util/lintpy.py -f
-    # XXX: Python lint checks are disabled until Issue #313 is resolved
-    condition: False
+      fork_origin="$(git merge-base --fork-point origin/master)"
+      git diff --name-only "$fork_origin" "util/*.py" \
```

Comment:
Our Python style guide applies to all Python code, not only `util` code. 

Please enable this lint for all Python code, and ignore failures for the moment (`continueOnError: true`). Please file a follow-up issue for the DV team to get their scripts lint clean (include the command to run), and another follow-up to disable the "dry run" mode.

I wouldn't do a PR to reformat large chunks of DV Python code at the moment with so much work ongoing in this domain, but feel free to do so for other areas where Python code is used.

Reply:
Ok, I'll make the change to the path. Understood about reformatting: my current plan is to make a "PEP8-fix" commit just before any proper patch to a file. That way, I can use flake8 linting to catch bugs in my own code, but I hopefully won't cause too much unnecessary churn. (That said, `git rebase` is usually excellent at sorting things out)